### ISSUE TITLE
Update of Services causes problem when having more than one Endpoint

### ISSUE THREAD

**Issue by slnetgit** [unchase/Unchase.Odata.Connectedservice#7]
If you have a Project with more than one ConnectedService it seems not to be possible to update the proxies after Generation so that the Right endpoint is taken for update.  The wizard Always only suggests the last created Service. Also custom namespaces seem to be set to Default after reopening the visualstudio. 
With this fixed your addin would be extremely useful because it supports entering of user credentials which the Microsoft one does not.

**Comment by KonstantinZikov** [comment]
Got the same issue when tried to update the proxy. In my case there were two services.

**Comment by slnetgit** [comment]
I made some Corrections to some of the files and with the corrections it works for me for odatav3 and vs2017 /2019.

[CorrectionsUnchase.zip](https://github.com/unchase/Unchase.Odata.Connectedservice/files/3461330/CorrectionsUnchase.zip)

the Problem has been that the wizard had read the data only from the settingsfile instead of - when updating - from the config file. also there had been a Problem when finishing the wizard immediately without visiting the advanced page when updating the Service. 
I also added the check for untrusted certificate when connecting to ssl Service .

I did not test this for odatav4 or the extensions because i do not Need this functionality at the Moment
